ee/codegen/src/__test__/nodes/generic-nodes/__snapshots__/generic-node.test.ts.snap

```diff
@@ -127,7 +127,7 @@ class MyCustomNode(BaseNode):
     default_attribute_2 = Inputs.count
 
     class NodeTrigger(BaseNode.Trigger):
-        merge_behavior = "AWAIT_ALL"
+        merge_behavior = MergeBehavior.AWAIT_ALL
```

Comment:
[blocking] wait, won't this start causing runtime errors because we're not importing `MergeBehavior`?